### ISSUE TITLE
github option doesn't work

### ISSUE THREAD

**Issue by Raynos** [nevir/groc#15]
```
  Publishing documentation to github...
! /usr/local/lib/node_modules/groc/scripts/publish-git-pages.sh: 10: Syntax error: "(" unexpected
```

Ubuntu 11.04

**Comment by nevir** [comment]
Doh, turns out I'm a complete sh newb.  I just pushed groc v0.2.5 (with 951c8dd27bd6150ebc7a5a17308941bb44426591), which should fix that issue at least

**Comment by nevir** [comment]
(this also points to the updated nevir/node-autorequire#2, which tackles the node version requirements that you were running up against over there)

**Comment by Raynos** [comment]
```
 Publishing documentation to github...
! /usr/local/lib/node_modules/groc/scripts/publish-git-pages.sh: 12: Syntax error: Bad for loop variable
```

**Comment by nevir** [comment]
Curses!  My hunch is that you have an actual `sh` and I'm doing bad things.  Rather than dealing with that, I'm just going to make bash a hard requirement for now: bd5c32ba6edaa543197be44f16bcd925856c8775

I pushed v0.2.6 which switches the shebang to point to bash instead of sh.  Does that clear it up?

**Comment by Raynos** [comment]
It ran but deleted all my files -.- `<not impressed>`

https://gist.github.com/1592084

**Comment by nevir** [comment]
You should be in the gh-pages branch - if so, run the following to get back into a good state:

```
git reset --hard HEAD && git checkout master
```

**Comment by nevir** [comment]
It should be only removing files checked into the `gh-pages` branch.  In order to figure out how that went wrong - what's your `.gitignore` look like?

**Comment by Raynos** [comment]
yeah I got the code back, it's just the script doesn't do anything useful :D,

Thankfully the script said "commit code before I continue running"

**Comment by nevir** [comment]
Yeah, definitely don't want to screw things up unless you have everything checked in

**Comment by nevir** [comment]
Do you have the `node_modules` directory checked into your project?  It looks like some files aren't owned by your user, so it's failing to clean those up (I don't want to inadvertently copy files around to your docs branch).  The script might be happy if you chown them all

**Comment by Raynos** [comment]
removing `node_modules` from git and adding it to `.gitignore` made the problem go away

**Comment by Raynos** [comment]
@nevir as an aside, is there any easy way to change the colour style of groc. (Your choice is ugly :()

**Comment by nevir** [comment]
@Raynos There isn't at the moment, but I'm in the process of revamping the style, I'll see about making that more extensible

**Comment by Raynos** [comment]
@nevir  also note that the menu doesn't build properly for me : http://raynos.github.com/nCore-example/test/server.html

**Comment by nevir** [comment]
Hmm, what browser are you using?

**Comment by Raynos** [comment]
@nevir never mind the bug is magically gone.
